### DISCUSSION TITLE
Having issues with Postgres constraint

### DISCUSSION THREAD

**Discussion by atzawada** [concourse/concourse#7230] [Help & Support]
Hi  all, I am having a problem when running a job. When performing a `get` step, the step completes successfully but at the end of the job I get the following output:
```
pq: there is no unique or exclusion constraint matching the ON CONFLICT specification
```

I am recovering an issue where the Postgres database went down and ever since I brought it back up the I have been getting this issue. Any help would be appreciated.

TIA

**Comment by taylorsilva** [comment]
What version of Concourse are you on?

**Reply by atzawada** [reply]
Hey sorry for not providing more details, I am on:
Concourse v7.0.0 installed via Helm deployment
Postgres 11 running on Ubuntu server

**Reply by taylorsilva** [reply]
In your logs do you see `failed-to-save-finish-get-event` at any point? I'm not sure where the error could be originating from exactly so taking a guess.

**Reply by atzawada** [reply]
In the ATC logs I am seeing `atc.tracker.notify.run.get-step.create.failed-to-initialize-cache` but not the one you have listed.

here are the relevant logs I believe:
```json
{"timestamp":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx","level":"error","source":"atc","message":"atc.tracker.notify.run.get-step.create.failed-to-initialize-cache","data":{"build":"29","build_id":2862679,"error":"pq: there is no unique or exclusion constraint matching the ON CONFLICT specification","job":"validate-pr","pipeline":"xxxxxxxxxxxxx","session":"23.1999.2.3.6","step-name":"merge-request","team":"xxxxxxxxxxxxx"}}
{"timestamp":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx","level":"info","source":"atc","message":"atc.tracker.notify.run.errored","data":{"build":"29","build_id":2862679,"error":"pq: there is no unique or exclusion constraint matching the ON CONFLICT specification","job":"validate-pr","pipeline":"xxxxxxxxxxxxx","session":"23.1999.2","team":"xxxxxxxxxxxxx"}}
{"timestamp":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx","level":"info","source":"atc","message":"atc.tracker.notify.run.finish.errored","data":{"build":"29","build_id":2862679,"error":"1 error occurred:\n\t* pq: there is no unique or exclusion constraint matching the ON CONFLICT specification\n\n","job":"validate-pr","pipeline":"xxxxxxxxxxxxx","session":"23.1999.2.8","team":"xxxxxxxxxxxxx"}}
```

**Comment by atzawada** [comment]
Didn't come to a good resolution on this, but upgrading minor versions to `7.3.2` seems to have fixed the problem for me.
